scala/scala_import.bzl

```diff
@@ -1,114 +1,81 @@
-#intellij part is tested manually, tread lightly when changing there
-#if you change make sure to manually re-import an intellij project and see imports
-#are resolved (not red) and clickable
+load(":providers.bzl", "JarsToLabels")
+
 def _scala_import_impl(ctx):
-    target_data = _code_jars_and_intellij_metadata_from(ctx.attr.jars)
```

Comment:
@andyscott why did you remove all the private methods? current code feels a bit like a wall of text